mitmproxy/addons/export.py

```diff
@@ -130,15 +137,23 @@ def raw(f: flow.Flow, separator=b"\r\n\r\n") -> bytes:
         and f.response
         and f.response.raw_content is not None
     )
+    is_websocket_flow = hasattr(f, "websocket") and f.websocket is not None
 
+    serialized_flow = b""
     if request_present and response_present:
-        return b"".join([raw_request(f), separator, raw_response(f)])
+        serialized_flow = serialized_flow.join(
+            [raw_request(f), separator, raw_response(f)]
+        )
```

Comment:
```suggestion
        serialized_flow = raw_request(f) + separator + raw_response(f)
```

and then remove `serialized_flow = b""` above. The existing code here is unidiomatic, let's fix that while we're at it. Thanks!